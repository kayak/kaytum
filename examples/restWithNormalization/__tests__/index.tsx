import * as React from "react";
import IndexPage from '../pages/index';
import {mount} from 'enzyme'

describe('IndexPage', () => {
  let page = null;

  beforeEach(() => {
    page = mount(<IndexPage />);
  });

  it('matches snapshot', () => {
    expect(page).toMatchSnapshot();
  });
});
