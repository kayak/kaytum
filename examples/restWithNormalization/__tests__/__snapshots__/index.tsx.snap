// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<Component>
  <Provider
    store={
      Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <TestComponent>
      <div>
        Loading user #
        1
        ...
      </div>
    </TestComponent>
  </Provider>
</Component>
`;
